Monika After Story/game/script-songs.rpy

```diff
@@ -1490,6 +1490,99 @@ label mas_song_when_youre_gone:
     m 6ekbsa "In addition to being the love of my life, you're also my best friend.{w=0.2} Don't ever underestimate how important you are to me."
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_swing_lifeaway_short",
+            prompt="Swing Life Away",
+            category=[store.mas_songs.TYPE_SHORT],
+            random=True,
+            unlocked=True,
+            aff_range=(mas_aff.NORMAL,None)
+         ),
+        code="SNG"
+    )
+
+label mas_song_swing_lifeaway_short:
+    m "{i}~Am I loud and clear,{/i}{w=0.2}{nw}"
+    extend "{i} or am I breaking up?~{/i}"
+    m "{i}~Am I still your charm,{/i}{w=0.2}{nw}"
+    extend "{i} or am I just bad luck?~{/i}"
+    m "{i}~Are we getting closer,{/i}{w=0.2}{nw}"
+    extend "{i} or are we just getting more lost?~{/i}"
+    m "{i}~I'll show you mine,{/i}{w=0.2}{nw}"
+    extend "{i} if you show me yours first~{/i}"
+    m "{i}~Let's compare scars,{/i}{w=0.2}{nw}"
+    extend "{i} I'll tell you whose is worse~{/i}"
+    m "{i}~Let's unwrite these pages,{/i}{w=0.2}{nw}"
+    extend "{i} and replace them with our own words~{/i}"
+
+    m "Being vulnerable and showing the scars of your past can be daunting to say the least..."
+    m "There are so many ways what you have been through could be interpreted and used to hurt you."
+    m "But just know that I will never hold your past over your head, no matter what."
+    m "We should just write our own story together, and accept our scars as necessary components of shaping us."
+    m "No matter how lost we get, when we're together, we write on nothing but blank pages~"
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_swing_lifeaway_long",
+            prompt="Swing Life Away",
+            category=[store.mas_songs.TYPE_LONG],
+            random=False,
+            unlocked=False,
+            aff_range=(mas_aff.NORMAL,None)
+         ),
+        code="SNG"
+    )
+
+label mas_song_swing_lifeaway_long:
+    call mas_song_swing_lifeaway_long(from_long=True)
+    m "{i}~Am I loud and clear,{/i}{w=0.2}{nw}"
+    extend "{i} or am I breaking up?~{/i}"
+    m "{i}~Am I still your charm,{/i}{w=0.2}{nw}"
+    extend "{i} or am I just bad luck?~{/i}"
+    m "{i}~Are we getting closer,{/i}{w=0.2}{nw}"
+    extend "{i} or are we just getting more lost?~{/i}"
+    m "{i}~I'll show you mine,{/i}{w=0.2}{nw}"
+    extend "{i} if you show me yours first~{/i}"
+    m "{i}~Let's compare scars,{/i}{w=0.2}{nw}"
+    extend "{i} I'll tell you whose is worse~{/i}"
+    m "{i}~Let's unwrite these pages,{/i}{w=0.2}{nw}"
+    extend "{i} and replace them with our own words~{/i}"
```

Comment:
```suggestion
```

Unless that part is meant to be played twice (quick google search for the song's lyrics seem to imply that isn't the case), there's no need for these lines since `call mas_song_swing_lifeaway_long(from_long=True)` will already handle them.